libpdbg/p10_fapi_targets.c

```diff
@@ -563,6 +563,16 @@ struct nx p10_nx = {
 };
 DECLARE_HW_UNIT(p10_nx);
 
+struct adc p10_adc = {
+    .target = {
+        .name = "POWER10 Analog to Digital Converter",
```

Comment:
Not sure this is right definition, ADC is not specific to p10 , system supports different types of ADCs' example ADC associated to APSS, and DIMM.

Reply:
I have added the abbreviation for the ADC as like previously added.

Please give some better names if you have a different view.